src/api/VoteMonitor.Api.PollingStation/Controllers/PollingStationV2Controller.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using AutoMapper;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using VoteMonitor.Api.PollingStation.Models;
+using VoteMonitor.Api.PollingStation.Queries;
+
+namespace VoteMonitor.Api.PollingStation.Controllers
+{
+    /// <summary>
+    /// Controller responsible for interacting with the polling stations - PollingStationInfo 
+    /// </summary>
+    [Route("api/v2/polling-station")]
+    public class PollingStationV2Controller : Controller
+    {
+        private readonly IMediator _mediator;
+        private readonly IMapper _mapper;
+
+        public PollingStationV2Controller(IMediator mediator, IMapper mapper)
+        {
+            _mediator = mediator;
+            _mapper = mapper;
+        }
+
+        [HttpGet]
+        [Produces(typeof(IEnumerable<GetPollingStation>))]
+        public async Task<IActionResult> GetAllPollingStations([FromQuery]PollingStationsFilter pollingStationsFilter)
+        {
+            //todo fix paging model defaults
```

Comment:
Please fix this :smile: It crashes with 500 now if the pagination params are missing from the request (`Page=1&PageSize=1`). It should the defaults in this case :smile:

Reply:
Ah yeah will do, sorry I noticed it but forgot to fix it

Reply:
Should be ok now :D